modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -468,47 +472,124 @@ TensorMechanicsAction::actOutputGeneration()
     {
       std::string type = "";
       InputParameters params = emptyInputParameters();
+      params = _factory.getValidParams("MaterialRealAux");
+      params.applyParameters(parameters());
+      params.set<MaterialPropertyName>("property") = _base_name + out;
+      params.set<AuxVariableName>("variable") = _base_name + out;
+      params.set<ExecFlagEnum>("execute_on") = EXEC_TIMESTEP_END;
+      _problem->addAuxKernel("MaterialRealAux", _base_name + out + '_' + name(), params);
+    }
+  }
+}
 
-      // RankTwoAux
-      for (const auto & r2a : _ranktwoaux_table)
+void
+TensorMechanicsAction::actOutputMatProp()
+{
+  std::string ad_prepend = "";
+  std::string ad_append = "";
+  if (_use_ad)
+  {
+    ad_prepend = "AD";
+    ad_append = "<RESIDUAL>";
+  }
+  //
+  // Create materials which allow the use of material properties for quatities
```

Comment:
Drop this comment -- it doesn't add much.

Reply:
it adds a typo...